src/systemtest/java/com/ericsson/ei/systemtests/FlowRunnerIT.java

```diff
@@ -0,0 +1,19 @@
+package com.ericsson.ei.systemtests;
```

Comment:
Why are we adding a Integrationtest flow runner in the systemtests?

Reply:
Name changed to ST

Reply:
To test reading the properties